azure-kusto-ingest/tests/e2e.py

```diff
@@ -144,14 +142,13 @@ def get_file_path() -> str:
 test_db = os.environ.get("TEST_DATABASE")
 
 python_version = "_".join([str(v) for v in sys.version_info[:3]])
-test_table = "python_test_{0}_{1}".format(python_version, str(int(time.time())))
+test_table = "python_test_{0}_{1}_{2}".format(python_version, str(int(time.time())), random.randint(1, 100000))
```

Comment:
GUID instead of random?

Reply:
I used random because I thought a UUID would be overkill. Aren't Python UUIDs 36 characters long? And wouldn't it take longer for Python to generate its 32 (not including dashes) random characters than to generate 5 random digits?

Reply:
I also considered using a random int *instead* of the python version and time components, but I think the 1st two fields are helpful to identify what (and when) created rogue Kusto tables.

Reply:
Probably, but is sends a clearer intent.
optional comment, not a must